src/main/java/by/custom/utilcalculator/controller/Bot.java

```diff
@@ -24,6 +24,7 @@ public static Bot getBot() {
     private Bot() {
         //initDB();
         messageRouter = MessageRouter.getInstance();
+        CommandTree.getInstance();
```

Comment:
Please, add a comment on why you call getInstance without initializing any variable like in the line above.

Reply:
Comment was added